lib/EarlyFunctionAnalysis/DetectABI.cpp

```diff
@@ -165,7 +165,7 @@ class DetectABI {
 private:
   CSVSet computePreservedCSVs(const CSVSet &ClobberedRegisters) const;
 
-  SortedVector<model::Register::Values>
+  TrackingSortedVector<model::Register::Values>
```

Comment:
Isolate this type of changes in its own commit

Reply:
made a commit containing changes to the generator and consequences to other parts of the code base